egg-herbie/main.rkt

```diff
@@ -22,7 +22,33 @@
                  [else "libegg_math"])
                 (bytes->string/utf-8 (system-type 'so-suffix)))))
 
-(define-ffi-definer define-eggmath (ffi-lib libeggmath-path))
+; Checks if Racket is being run in emulation via rosetta
+(define (check-for-rosetta)
+  (equal? (with-output-to-string (lambda () (system "sysctl -n sysctl.proc_translated"))) "1\n"))
+
+(define fallback-message
+  "Error: unable to load the 'egg-math' library.")
+
+; Note, refering to ARM as Apple Silicon to match Racket download page.
+(define rosetta-message
+  (string-join
+   '("You are trying to run 'Herbie' with the 'x86' version of 'Racket' under emulation,"
+     "via 'Rosetta', Please install the the 'Apple Silicon' version of 'Racket'."
+     "\"https://download.racket-lang.org\"")

```

Comment:
```suggestion
   '("You are running the 'x86' version of Racket via 'Rosetta' emulation."
     "To run Herbie, you need to use the 'Apple Silicon' version of Racket."
     "You can install it here: https://download.racket-lang.org")
```

I wouldn't put Herbie and Racket in quotes. Right now, users have to manually install both, so they probably know what they are. I like the quotes on Rosetta, x86, and Apple Silicon. Also, make sure each line is under 72 characters; I think that's the case with the suggestion above.